index3.js

```diff
@@ -0,0 +1,104 @@
+let max = 100;
+//gives you a random number in a range as solid integers (in this case 1-100)
+
+initialize();
+
+async function initialize() {
+  console.log("Which game would you like to play?");
+
+  let game = await ask(
+    "Would you like to pick or guess the number? \nType 'guess' or 'pick'. \n"
+  );
+  if (game === "guess") {
+    guess();
+  }
+  if (game === "pick") {
+    pick();
+  } 
+}
+
+async function guess() {
+  /*console.log(
+    "Let's play a game where I (computer) make up a number and you (human) try to guess it."
+  );*/
+  let secretNumber = randomNum(Math.floor(min), Math.floor(max));
+  //console.log(secretNumber);
+  //The above console.log is a check to confirm that you guessed the correct number.
+
+  let myGuess = await ask(
+    "Ok ready? Guess a whole number between " + min + " and " + max + ".\n"
+  );
+
+  if (myGuess === secretNumber) {
+    console.log("Yes! You win!");
+    process.exit();
+  }
+  //Starts a loop of options if you don't guess the secret number on the first try.
+  while (myGuess !== secretNumber) {
+    if (myGuess > secretNumber) {
+      myGuess = await ask("No, guess lower\n");
+    }
+    if (myGuess < secretNumber) {
+      myGuess = await ask("No, guess higher\n");
+    }
+    if (myGuess === secretNumber) {
+      console.log("Yes! You win!");
+    }
+  }
+}
+
+async function pick() {
+  console.log(
+    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
+  );
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+  console.log("You entered: " + secretNumber);
+  // Now try and complete the program.
+
+  let computerGuess = randomNum(min, max);
+  console.log("I guess " + computerGuess);
+
+  let answer = await ask("Is this the correct number?");
+
+  //If the computer guesses the correct number, the user answers "yes". Then "You win!" And then the program exits.
+  if (answer.toLowerCase() === "yes") {
+    console.log("You win!");
+    process.exit();
+  }
+  //If the computer doesn't guess the correct number, user responds "no".
+  while (answer.toLowerCase() === "no") {
+    let negAnswer = await ask("Is the number higher or lower?"); //Then the question of "higher or lower?", followed by the appropriate answer by user.
+    if (negAnswer.toLowerCase() === "higher") {
+      min = computerGuess + 1;
+    }
+    if (negAnswer.toLowerCase() === "lower") {
+      max = computerGuess - 1;
+    }
+    //cheater function
+    if (min > max || max < min) {
+      console.log("You're cheating!");
+      process.exit();
+    }
+    computerGuess = Math.floor((min + max) / 2); //The computer guesses again, but using the binary search algorithm to maximize guess efficiency.
+    console.log("I guess " + computerGuess);
+    answer = await ask("Is this the correct number?"); //Then asks the user to confirm their answer.
+    if (answer.toLowerCase() === "yes") {
+      console.log("You win!\n");
+    }
```

Comment:
As I said up in index1.js, there is no exit here so the game gets stuck at this point.